src/Kunstmaan/AdminBundle/Helper/Menu/SettingsMenuAdaptor.php

```diff
@@ -15,18 +15,24 @@
 class SettingsMenuAdaptor implements MenuAdaptorInterface
 {
     /**
-     * @var ContainerInterface
+     * @var AuthorizationCheckerInterface
      */
-    private $container;
+    private $authorizationChecker;
+
+    /**
+     * @var boolean
+     */
+    private $isEnabledVersionChecker;
 
     /**
      * Constructor
      *
-     * @param ContainerInterface $container
+     * @param AuthorizationCheckerInterface $container
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(AuthorizationCheckerInterface $authorizationChecker, $isEnabledVersionChecker)
```

Comment:
This PR wasn't marked as a BC but since it changes the constructor of a service that can be extended, it seems like it is. We're extending this class and passing the container to the parent constructor, now causing it to throw a fatal.

Reply:
@wesleylancel it's been done against the 5.0 branch, this one is for BC. I'll change the PR description.

Reply:
@sandergo90 Should be in the UPGRADE doc too, that's what we normally check.

Reply:
@wesleylancel You're correct. @cv65kr Can you change the upgrade guide for the PR ?